Dangerfile

```diff
@@ -0,0 +1,12 @@
+def fail_if_no_supported_label_found
+  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test", "next_release"]
+
+  supported_labels_in_pr = supported_types & github.pr_labels
+  no_supported_label = supported_labels_in_pr.empty?
+  if no_supported_label
+    fail("Label the PR using one of the change type labels: #{supported_types}")
+  end
+end
+
+fail_if_no_supported_label_found
+  
```

Comment:
Hmm this is weird: 
![image](https://user-images.githubusercontent.com/808417/185398411-4ca10b5c-7352-42a8-b587-3061e114df2e.png)
Usually that symbol should be above... Not sure what it means here 🤷‍♂️

Reply:
🤔 I have no idea haha but I tried removing the last line and re-adding it and Git didn't detect any change so I am not sure

Reply:
ok, it was an extra space lol